src/store/modules/app/actions.js

```diff
@@ -147,11 +155,12 @@ const actions = {
   },
   async changeOSKeyboardLayout({ dispatch, state, commit }, osLayout) {
     commit('setOSKeyboardLayout', osLayout);
-    // Important to call keycodes/updateKeycodeNames *before* keymap/updateKeycodeNames.
-    this.commit('keycodes/updateKeycodeNames');
+    const keycodesStore = useKeycodesStore();
+
+    // Important to call keycodes/updatekeycodeNames *before* keymap/updateKeycodeNames.
```

Comment:
```suggestion
    // Important to call keycodesStore.updateKeycodeNames *before* keymap/updateKeycodeNames.
```
Updating the comment to reflect changes to the call signature